PhysicsTools/PatAlgos/python/slimming/prunedGenParticles_cfi.py

```diff
@@ -15,6 +15,7 @@
         "drop   status == 2 && abs(pdgId) == 21",                          # but remove again gluons in the inheritance chain
         "keep abs(pdgId) == 23 || abs(pdgId) == 24 || abs(pdgId) == 25 || abs(pdgId) == 6 || abs(pdgId) == 37 ",   # keep VIP(articles)s
         "keep abs(pdgId) == 310 && abs(eta) < 2.5 && pt > 1 ",                                                     # keep K0
+        "+keep abs(pdgId) == 13 && status == 1", # keep muon parents
```

Comment:
@slava77: @gpetruc could confirm, but from what I see is that the inclusive statement you are referring to does not seem to pick up the pion and Kaon parents of muons and one has to specify explicitly the status.